lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -122,6 +119,20 @@ C<text> is the textual value to be presented in the UI.
 
 has selectable_values => (is => 'ro', isa => 'HashRef', required => 0);
 
+sub _displayed_key {
```

Comment:
The name suggests this returns displayed keys. However, from reading the code, that's not what it seems to do.

Do we really need this check? If so, maybe add a bit of documentation using `#` comments why this check is invaluable and what the exact check is achieving?

Reply:
Renamed. Selectable values must be displayed otherwise test can't succeed.